docs/site/translation.md

```diff
@@ -392,3 +392,7 @@ entry you're editing/vetting. Use it if in doubt.*
 [Year First Calendar]: /translation/date-time/date-time-patterns#year-first-calendar
 [Example Hidden]: https://st.unicode.org/cldr-apps/v#/USER/Number_Formatting_Patterns/67afe297d3a17a3
 [Key for Show Hidden]: https://cldr.unicode.org/translation/core-data/exemplars#key-to-escapes
+[Unicode Characters on Mac]: https://ladedu.com/how-to-enter-unicode-characters-on-a-mac
```

Comment:
These need to be moved to the Unicode Sets page. Currently the links don't work.

Reply:
done